src/query/api/v1/handler/namespace/common.go

```diff
@@ -128,40 +122,39 @@ func RegisterRoutes(
 	}
 
 	// Get M3DB namespaces.
-	getHandler := wrapped(
-		applyMiddleware(NewGetHandler(client, instrumentOpts).ServeHTTP, defaults))
-	r.HandleFunc(M3DBGetURL, getHandler.ServeHTTP).Methods(GetHTTPMethod)
+	getHandler := applyMiddleware(
```

Comment:
nit; we apply middleware to all of these, can we roll this step into the addRoute function passed into here?